src/exporters/stakingMinerAccount.ts

```diff
@@ -0,0 +1,43 @@
+import * as PromClient from "prom-client"
+import { decimals } from '../utils';
+import { logger } from '../logger';
+import { Exporter } from './IExporter';
+import { ApiPromise, WsProvider } from "@polkadot/api";
+import { Header } from "@polkadot/types/interfaces";
+import { StakingMinerAccount } from '../workers/stakingMinerAccountWorker'
+import { STAKING_MINER_ACCOUNT_WORKER_PATH } from '../workers/workersPaths'
+
+class StakingMinerAccountExporter extends StakingMinerAccount implements Exporter {
+    palletIdentifier: any;
+    registry: PromClient.Registry;
+
+    constructor(registry: PromClient.Registry) {
+        super(STAKING_MINER_ACCOUNT_WORKER_PATH, registry, true);
+        this.registry = registry;
+        this.palletIdentifier = "system";
+
+    }
+
+    async init(api: ApiPromise, chainName: string, startingBlockTime: Date, endingBlockTime: Date) {
+
+        await this.clean( api, chainName.toString(), startingBlockTime, endingBlockTime);
+
+    }
+
+    async perBlock(api: ApiPromise, header: Header, chainName: string): Promise<void> {
+
+        const blockNumber = parseInt(header.number.toString());
+        const result = await this.doWork(this,api,  blockNumber, chainName)
+ 
+    }
+
+    async perDay(api: ApiPromise, chainName: string) { }
```

Comment:
you might want to choose to collect only once per day or per hour metrics that take a lot of time to compute and to request, so this is the purpose of these functions. If you write a metric in the per hour, you should not write it in the perBlock section. But those 3 functions concern the real time only. When you load history, there is going to be the possibility to specify the distance between blocks, so it will be aligned with the ongoing data collection.